### ISSUE TITLE
[CDC] Eliminate unnecessary logging when cdc is not enabled

### ISSUE THREAD

**Issue by ajcaldera1** [yugabyte/yugabyte-db#4246]
Jira Link: [DB-2162](https://yugabyte.atlassian.net/browse/DB-2162)
When CDC is not enabled, tserver keeps checking for the minimum replicated indices and failing to find the cdc_state table. It might be a good idea to either suppress the warning or always create cdc_state in the system catalog. 
``` 
I0416 02:45:12.281244 11940 cdc_service.cc:458] Started to read minimum replicated indices for all tablets
W0416 02:45:12.282356 11961 client-internal.cc:1203] GetTableSchemaRpc(table_identifier: table_name: "cdc_state" namespace { name: "system" database_type: YQL_DATABASE_CQL }, num_attempts: 1) failed: Not found (yb/master/catalog_manager.cc:3529): The object does not exist: table_name: "cdc_state" namespace { name: "system" database_type: YQL_DATABASE_CQL }
```

**Comment by hectorgcr** [comment]
We currently have no way to detect if CDC is disabled for all tablets. We can use an non-existent or empty `cdc_state` table to infer this.

**Comment by bmatican** [comment]
Potentially a good usability task for new folks @nspiegelberg

**Comment by joe-maley** [comment]
Fixed by 7852d548c0a0de90f498027a84abead9289e1554. Closing this issue.

The check for minimum replicated indices runs on the `update_peers_and_metrics_thread_`. The referenced commit made a change to skip the work on this thread when CDC is disabled:
```
void CDCServiceImpl::UpdatePeersAndMetrics() {
  ...
  do {
    if (!cdc_enabled_.load(std::memory_order_acquire)) {
      // CDC service not enabled, so skip background thread work.
      continue;
    }
    ...
    YB_LOG_EVERY_N_SECS(INFO, 300)
      << "Started to read minimum replicated indices for all tablets";
    ...
  }
}
```
